Api/SplinterlandsAPI.cs

```diff
@@ -134,20 +134,21 @@ public static async Task<(int newRating, int ratingChange, decimal spsReward, in
                 {
                     gameResult = 2;
                 }
-
+var rhsr = JToken.Parse((string)matchHistory["battles"][0]["dec_info"]);
                 int newRating = (string)matchHistory["battles"][0]["player_1"] == username ? ((int)matchHistory["battles"][0]["player_1_rating_final"]) :
                     ((int)matchHistory["battles"][0]["player_2_rating_final"]);
                 int ratingChange = (string)matchHistory["battles"][0]["player_1"] == username ? newRating - ((int)matchHistory["battles"][0]["player_1_rating_initial"]) :
                     newRating - ((int)matchHistory["battles"][0]["player_2_rating_initial"]);
                 decimal spsReward = (decimal)matchHistory["battles"][0]["reward_sps"];
+				int glint = Convert.ToInt32((int)rhsr["glint"]);
 
-                return (newRating, ratingChange, spsReward, gameResult);
+                return (newRating, ratingChange, spsReward, gameResult,glint);
```

Comment:
i would suggest a space between the comma and glint for readability and to match style